versioned_docs/version-0.4.2/tutorials/common/_starting-dashboard.mdx

```diff
@@ -10,7 +10,7 @@ The current version of the dashboard is ready to use and available
 [here](https://jellyfish-dev.github.io/jellyfish-dashboard/).
 Ensure that it is compatible with your Jellyfish server!
 Please note that this dashboard only supports secure connections (https/wss)
-or connections to localhost. Any insecure requests (http/ws) will be automatically blocked by the browser.
+or connections to localhost. Any insecure requests (http/ws) will be automatically blocked by the browser. For testing purposes we recommend to use Chrome browser. Otherwise without secure connection you may encounter problems with connecting to Jellyfish server.
```

Comment:
```suggestion
or connections to localhost. Any insecure requests (http/ws) will be automatically blocked by the browser. For testing purposes, we recommend to use Chrome browser. Otherwise you may encounter problems with connecting to Jellyfish server without secure connection.
```